scene/2d/collision_object_2d.cpp

```diff
@@ -56,15 +57,15 @@ void CollisionObject2D::_notification(int p_what) {
 			_update_pickable();
 
 			//get space
-		}
+		} break;
```

Comment:
Here too, I added `break:`s as the fallthrough did not seem intentional, but if it is, this change might cause bugs.